pkgs/by-name/su/surfer/package.nix

```
@@ -0,0 +1,56 @@
+{
+  lib,
+  fetchFromGitLab,
+  rustPlatform,
+  pkg-config,
+  openssl,
+  wayland,
+  libxkbcommon,
+  libGL,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "surfer";
+  version = "v0.2.0";
+
+  src = fetchFromGitLab {
+    owner = "surfer-project";
+    repo = pname;
```

```suggestion
    repo = "surfer";
```
it's bad practice to conflate the package name and repository name. see https://github.com/NixOS/nixpkgs/issues/277994